soroban-env-common/src/env.rs

```diff
@@ -96,6 +96,12 @@ pub trait EnvBase: Sized + Clone {
         vals: &[RawVal],
         strs: &[&'static str],
     ) -> Result<(), Status>;
+
+    /// Fill a byte slice from the host PRNG.
+    fn prng_fill_slice(&self, _mem: &mut [u8], _finalize: bool) -> Result<(), Status> {
+        // TODO: remove default impl when SDK env implements prng_fill_slice.
+        unimplemented!()
```

Comment:
💡 I think it's okay if you want to drop the default impl here, and we'll just impl this straight away in the SDK. I don't mind though. If you leave this out it will make sure we don't miss impl this on the SDK 😄

Reply:
I can't recompile the examples without it (they use the SDK).